docs/_v15_x/2_send.md

```diff
@@ -255,6 +255,87 @@ client.createMessage(message)
         .send();
 ```
 
+### Datatype ShareDocumentsRequest
+
+This datatypes facilitates exchange of documents between an organisation and a Digipost end user. The sender
+first sends a message of datatype ShareDocumentsRequest, to which the end user can attach a list of documents. When
+new documents are shared, a DocumentEvent is generated. The organisation can retrieve the status of their
+ShareDocumentsRequest. If documents are shared and the sharing is not cancelled, the documents can either be downloaded
+or viewed on the digipostdata.no domain. Active requests can be cancelled both by the end user and the organisation.
+
+The `purpose` attribute of the ShareDocumentsRequest should briefly explain why the sender organisation want to gain
+access to the relevant documents. The primary document should contain a more detailed explanation.
```

Comment:
Mulig vi burde vektlegge enda mer at dette er den juridiske grunnen til at virksomheten ønsker å få innsyn i dokumentet. Kanskje vår personvern-ekspert eller sikkerhetsansvarlig har en god formulering på lur? @sarawe @hanskhe

Reply:
Kanskje legge til noe sånn som: "This text will be displayed prominently, and should contain the information necessary for the user to make an informed choice." før den siste setningen?